.github/workflows/release.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             target: x86_64-unknown-linux-gnu
             binary: x86_64-manylinux2014-rocm5.7
             container: rocm/dev-ubuntu-22.04:5.7-complete
-            apt_dependencies: git pkg-config libssl-dev cmake protobuf-compiler
```

Comment:
this is usually handled in bash ./ci/prepare_build_environment.sh

Reply:
Well I don't have access to the type of the image from there, so not sure how I could do it from there.

Reply:
Ah and the script also runs too late, because it runs after the checkout, but then it'll fail because the git is not installed

Reply:
Should probably to be fixed with the manylinux image anyway, as soon as it builds.

Reply:
Kinda solved